policyengine_us/parameters/gov/ed/pell_grant/amount/max.yaml

```
@@ -0,0 +1,9 @@
+description: Maximum value of the Pell Grant
```

+ other descriptions
```suggestion
description: The Department of Education limits Pell Grants to this amount.
```